cardInfoAPI-Level1.yaml

```
@@ -0,0 +1,1402 @@
+
+  /cards/{cardId}:
+    get:
+      summary: Provides a single card which matches the id.
+      description: >
+        This operation enables the retrieval of a single payment card when the cardId is already known.
+        The cardId must be retrieved with the GET /cards operation before or stored locally at the API consumer.
+      tags:
+        - Cards
+      parameters:
+        - $ref: '#/components/parameters/path_cardId'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Single payment card was found. If the cardId was not found, a 400 is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CardDetails'
+        '400':
+          $ref: '#/components/responses/standard400'
+        '500':
+          $ref: '#/components/responses/standard500'
+
+  /card-tokens:
+    get:
+      summary: Provides a list of card tokens which are associated with a card.
+      description: >
+        This operation enables the retrieval of a list of tokens associated with a payment card.
+        The tokens can be filtered further by status.
+        Tokens are representations of a payment card used in specific wallets , wearables or merchants.
+      tags:
+        - Card-tokens
+      parameters:
+        - in: query
+          name: card_id
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardId'
+          description: Retrieves all tokens related to a card. Value has to be a technical identifier for the card that can be retrieved with the GET /cards endpoint.
+        - in: query
+          name: token_status
+          required: false
+          schema:
+            $ref: '#/components/schemas/TokenStatus'
+          description: Retrieves all tokens in a certain status. Only one status can be filtered at a time.
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: >
+            Card tokens were found which match the query parameters.
+            If no card tokens matched the parameters or access rights are missing, an empty list is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardList:
```

Should be the same name as the API resource (GET /card-tokens) or the resourename is wrong.
```suggestion
                  cardTokens:
```

---

Thanks, will be adjusted.